Meter.c

```diff
@@ -333,6 +334,10 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
       GraphMeterMode_pixPerRow = PIXPERROW_ASCII;
    }
 
+   if ((size_t)w * 2 > nValues) {
+      x += w - (nValues / 2);
```

Comment:
Can you drop those superfluous parens?
There are some more places, where they aren't strictly necessary and may be removed.

Reply:
This one casts from a signed type (`w`) to unsigned (`nValues`). I'll get a compile warning if I don't make the explicit cast.